RELEASE.md

```diff
@@ -0,0 +1,65 @@
+# Release
+
+Releases are mostly automated using
+[release-it](https://github.com/release-it/release-it/) and
+[lerna-changelog](https://github.com/lerna/lerna-changelog/).
+
+
+## Preparation
+
+Since the majority of the actual release process is automated, the primary
+remaining task prior to releasing is confirming that all pull requests that
+have been merged since the last release have been labeled with the appropriate
+`lerna-changelog` labels and the titles have been updated to ensure they
+represent something that would make sense to our users. Some great information
+on why this is important can be found at
+[keepachangelog.com](https://keepachangelog.com/en/1.0.0/), but the overall
+guiding principles here is that changelogs are for humans, not machines.
```

Comment:
principles (plural) here ~~is~~ are

Reply:
This is auto generated. I just opened a PR to fix it https://github.com/rwjblue/create-rwjblue-release-it-setup/pull/12